.travis.yml

```diff
@@ -9,12 +12,18 @@ python:
   - "2.7"
   - "3.5"
 
+env:
+  - TEST_DIR=root TEST_TYPE=lint
+  - TEST_DIR=root TEST_TYPE=unit
+  - TEST_DIR=root TEST_TYPE=functional
+  - TEST_DIR=utils
+
 install:
   - pip install -r requirements.txt
   - pip install tox-travis
 
 script:
   # TODO(rhcarvalho): check syntax of other important entrypoint playbooks
   - ansible-playbook --syntax-check playbooks/byo/config.yml
-  - tox
-  - cd utils && tox
+  - if [[ "${TEST_DIR}" == "root" ]]; then tox; fi
+  - if [[ "${TEST_DIR}" == "utils" ]]; then cd utils && tox; fi
```

Comment:
Agreed, mainly was trying to cleanup travis output to avoid the utils tox tests being the last result summary shown.

I still want to break them up into separate runs, but I agree it can be done much cleaner.